packages/scripts/src/entry.mts

```diff
@@ -66,21 +65,23 @@ async function bootstrap(command: string) {
 		try {
 			await loadCommand(`_${command}.mjs`)
 		} catch (err) {
+			console.error(err)
 			error(
-				`unknown command "${command}".\nSee --help for a list of available commands.`,
+				`could not load command command "${command}".\nSee --help for a list of available commands.`,
 			)
+			process.exit(1)
 		}
 	} else {
 		await loadAllCommands()
 	}
 
 	// error on unknown commands
 	program.on('command:*', () => {
-		console.log(
-			'Invalid command: %s\nSee --help for a list of available commands.',
-			program.args.join(' '),
-		)
-		process.exit(1)
+		// console.log(
```

Comment:
Did you mean to leave this commented out?

Reply:
I meant to delete it; pushing an update